### ISSUE TITLE
Ratio of removed nodes for inductive experiment

### ISSUE THREAD

**Issue by giang-n-ngo** [hwwang55/PathCon#12]
Section 4.2 describes an experiment for inductive KG completion which includes sampling a subset of nodes in the test set and removing them from the training set.
How was this implemented in the code? I am looking at data_loader.py, but I cannot seem to find it.
Thank you.
